### ISSUE TITLE
Release new update

### ISSUE THREAD

**Issue by ametad** [fzaninotto/Faker#1499]
Hi fzaninotto,

First of all thank you for all the work you've done! I really like this package (Faker) of yours. 
Some months ago I pushed code to this repo (https://github.com/fzaninotto/Faker/pull/1323). 

I was hoping perhaps you would release a new version so I can make use of fake data that was pushed here.

Kind regards.

**Comment by Einenlum** [comment]
:+1: 

It seems the last tag right now (1.7.1) is 203 commits behind master.

https://github.com/fzaninotto/Faker/compare/v1.7.1...master

**Comment by ametad** [comment]
@ppelgrims Perhaps none of my business, but are you asking me or the owner of this repository?

**Comment by ppelgrims** [comment]
Anyone can contribute, this is open source. I know François is very busy at the moment so it's more likely to get done or done faster when you contribute.

**Comment by ametad** [comment]
Good point ppelgrims! I try to contribute, but I also am busy at the moment. But when I can squeeze some I will.

**Comment by ametad** [comment]
Is it the intention that all commits are mentioned and from whom they come?

**Comment by ppelgrims** [comment]
It looks like it's done per PR, not per commit.
[CHANGELOG.md](https://github.com/fzaninotto/Faker/blob/master/CHANGELOG.md)

**Comment by fzaninotto** [comment]
I use a package to generate the changelog automatically, I'll try to do it soon.

**Comment by ametad** [comment]
Great!
Francois Zaninotto you are the best :) Thank you!
